storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedOperation.java

```diff
@@ -62,4 +62,7 @@
     @NonNull
     @CheckResult
     Single<Result> asRxSingle();
+
+    @NonNull
+    Data getData();
```

Comment:
So basically this does not help interceptors to be type-safe but gives a way to get data without need in casting `PreparedOperation` to some particular implementation.

I guess we'll have to ship it as 2.x then since it's addition to an interface which is a breaking change (when I was giving greenlight to that I was thinking just about `, Data>` generic which is not breaking but forgot that it'll add a method)